rainbow-me/browser-extension
# feat: uses current theme from theme store

Fixes BX-161
Figma link (if any):

## What changed (plus any additional context for devs)
- Uses theme from `useCurrentThemeStore()`
- `ThemeProvider` only accepts `dark` or `light` but users "see" (`light`, `dark`, `system`), so we probably need to keep some state that we use for "display" vs state that we use to pass to design system


## Screen recordings / screenshots



## What to test
- In Settings, change the theme
- Close and open the extension to see if the theme persists
- Set theme to "system" and see if it respects that setting
- Adjust system light/dark mode, see if changes reflected when switching to new "system" mode.


## Final checklist
